From 5f5a7c78f42713db3ac6fe95c0445f0ff1f78968 Mon Sep 17 00:00:00 2001
From: nucleo <nucleo@fedoraproject.org>
Date: Thu, 26 Feb 2026 15:57:47 +0200
Subject: [PATCH] xdpdump: include <limits.h> removed from linux/ethtool.h

Removing include <limits.h> from linux/ethtool.h resulted
in a undeclared UINT_MAX and ULONG_MAX compilation errors

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=a8a11e5237aed71b7f5f9d33c554ef06fe974311

Added #include <limits.h> in xdp-dump/xdpdump.c

Signed-off-by: nucleo <nucleo@fedoraproject.org>
---
 xdp-dump/xdpdump.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xdp-dump/xdpdump.c b/xdp-dump/xdpdump.c
index 19d51e4..fa3aeb3 100644
--- a/xdp-dump/xdpdump.c
+++ b/xdp-dump/xdpdump.c
@@ -7,6 +7,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <inttypes.h>
+#include <limits.h>
 #include <signal.h>
 #include <stdbool.h>
 #include <stdio.h>
-- 
2.53.0

